src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -182,6 +182,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleHelp();
             }
 
+            if (commandResult.anyFunctionToRun()) {
```

Comment:
should the name be something like "hasFunctionToRun"?

Reply:
i think in a javascript world, it is called a callback. 

Perhaps `hasCallback` and `getCallback` is a better name?

Reply:
Updated